src/Utils/util.ts

```diff
@@ -93,4 +94,10 @@ export function earlierDateOrTimeToday(timestamp: string): string {
     const dialogTime = moment(timestamp)
     const isDialogCreatedToday = dialogTime.diff(endOfYesterday) >= 0
     return dialogTime.format(isDialogCreatedToday ? 'LTS' : 'L')
+}
+
+export function isNewActionUnique(new_action: CLM.ActionBase, actions: CLM.ActionBase[]): boolean {
+    const needle = stringify({ ...new_action, actionId: "", createdDateTime: "" })
```

Comment:
Optional, but instead of forcing the properties you want to ignore to be the same (empty string) on newAction and other actions I would think it makes more sense to remove them.  Seems to better convey intent of NOT comparing these properties since they are expected to be different.

Reply:
good suggestion.